### DISCUSSION TITLE
asus vivobook r7 6800H  smt

### DISCUSSION THREAD

**Discussion by max-ing** [DavidS95/Smokeless_UMAF#49] [Q&A]
There is no option to turn off hyperthreading (smt) .Device Manager > AMD CBS > CPU Common Options .

**Comment by kalimaenergy** [comment]
Same on G513QY with 5900HX

**Comment by kalimaenergy** [comment]
@DavidS95 Hello, i don't know if i can help you with some informations or dump for make this available ?

Thank you.
